content/news/buzz/march-2023-status-update/contents.lr

```diff
@@ -0,0 +1,51 @@
+title: March 2023 Status Update
+---
+author: Russell Keith-Magee
+---
+body:
+
+March has been a month of slow but steady progress for BeeWare.
+
+What we've done
+----------------
+
+* Progress on GUI testing in Toga is continuing. We now have 7 widgets fully tested, with another 2 in progress. As with last month's update, getting these tests passing has required wrestling a lot of fundamentals, in particular around the simulation of events such as mouse clicks. We're also discovering (and fixing) lots of corner cases in widget implementations that only surface when you're doing rigorous and repeatable testing.
+* We've made some `major changes to the way GTK widgets are rendered <https://github.com/beeware/toga/pull/1794>`__. This change signficantly reduces the amount of re-rendering that is performed by GTK, as well as resolving a number of long standing issues related to resizing widgets.
+* Thanks to some community contributions, we were were able to add support for `focus events and text alignment <https://github.com/beeware/toga/pull/1817>`__ on GTK's TextInput widget, and support for `text alignment and fonts <https://github.com/beeware/toga/pull/1821>`__ on GTK's NumberInput widget.
+* We landed the implementation of a `Linux System package backend <https://github.com/beeware/briefcase/pull/1106>`__ for Briefcase. Thanks to a community contribution, we were also able to add support for `Arch packaging <https://github.com/beeware/briefcase/pull/1143>`__
+* We modified the `locations where Briefcase outputs build artefacts <https://github.com/beeware/briefcase/pull/1107>`__. This removes some sources of bugs associated with tools not supporting spaces in filenames, and makes Briefcase's output consistent with other tools in the Python ecosystem.
+* We made significant progress on `modifying Briefcase's AppImage and Flatpak backends <https://github.com/beeware/briefcase/pull/1132>`__ to use Indygreg's `Standalone Python <https://github.com/indygreg/python-build-standalone>`__, rather than BeeWare's Linux support package. Standalone Python is becoming a defect standard for pre-compiled Python builds (especially on Linux), so it makes sense for BeeWare to converge on this common standard. This change will also significantly speed up Flatpak builds, should be more stable on AppImage, allows us to switch to `manylinux <http://github.com/pypa/manylinux>`__ base images - and it reduces the number of projects that BeeWare is responsible for maintaining. We're currently waiting on a `fix to the linuxdeploy GTK plugin <https://github.com/linuxdeploy/linuxdeploy-plugin-gtk/pull/45>`__ to be merged before we can land this change.
```

Comment:
```suggestion
* We made significant progress on `modifying Briefcase's AppImage and Flatpak backends <https://github.com/beeware/briefcase/pull/1132>`__ to use Indygreg's `Standalone Python <https://github.com/indygreg/python-build-standalone>`__, rather than BeeWare's Linux support package. Standalone Python is becoming a de facto standard for pre-compiled Python builds (especially on Linux), so it makes sense for BeeWare to converge on this common standard. This change will also significantly speed up Flatpak builds, should be more stable on AppImage, allows us to switch to `manylinux <http://github.com/pypa/manylinux>`__ base images - and it reduces the number of projects that BeeWare is responsible for maintaining. We're currently waiting on a `fix to the linuxdeploy GTK plugin <https://github.com/linuxdeploy/linuxdeploy-plugin-gtk/pull/45>`__ to be merged before we can land this change.
```